larpandoracontent/LArHelpers/LArGeometryHelper.cc

```diff
@@ -138,7 +138,7 @@ void LArGeometryHelper::MergeTwoPositions(const Pandora &pandora, const HitType
     const float Z3(LArGeometryHelper::MergeTwoPositions(pandora, view1, view2, Z1, Z2));
 
     position3.SetValues(X3, 0.f, Z3);
-    const float sigmaUVW(pandora.GetGeometry()->GetLArTPC().GetSigmaUVW());
+    const float sigmaUVW(LArGeometryHelper::GetSigmaUVW(pandora));
```

Comment:
Should be a single float (as in LArTPC class), except when used in mathematical manipulation in RotationalTransformationPlugin.